sysbuild/Kconfig.tfm

```diff
@@ -0,0 +1,38 @@
+# Copyright (c) 2024 Nordic Semiconductor
```

Comment:
The provisioning is done for the MCUboot, so it does not see the Kconfigs of the application. It might be possible to change this to include Kconfigs of the application, but whether this makes sense would need input from @nordicjm.

Reply:
sysbuild files go in sysbuild, this is the correct folder